acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/configuration/BesuNodeConfigurationBuilder.java

```diff
@@ -167,8 +167,7 @@ public BesuNodeConfigurationBuilder jsonRpcAuthenticationConfiguration(final Str
     return this;
   }
 
-  public BesuNodeConfigurationBuilder jsonRpcAuthenticationUsingPublicKeyEnabled()
-      throws URISyntaxException {
+  public BesuNodeConfigurationBuilder jsonRpcAuthenticationUsingRSA() throws URISyntaxException {
     final File jwtPublicKey =
         Paths.get(ClassLoader.getSystemResource("authentication/jwt_public_key").toURI())
```

Comment:
can we rename this one to jwt_publik_key_rsa

Reply:
or even rs256

Reply:
b27b7e7